.github/workflows/release.yml

```diff
@@ -30,20 +28,6 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ github.token }}
 
-    - name: create GitHub release (canary)
-      if: github.ref == 'refs/heads/main'
-      run: |
-        gh release delete canary --cleanup-tag
-        gh release create canary dist/spin-plugin-kube*.tar.gz \
-          --title canary \
-          --prerelease \
-          --notes-file - <<- EOF
-          This is a "canary" release of the most recent commits on our main branch. Canary is **not stable**.
-          It is only intended for developers wishing to try out the latest features in Spin, some of which may not be fully implemented.
-        EOF
-      env:
-        GH_TOKEN: ${{ github.token }}
-
     - name: Release Plugin
       uses: rajatjindal/spin-plugin-releaser@v1
```

Comment:
we probably need to modify logic here to run the releaser action only on tagged releases?

Reply:
Wouldn’t the changes to lines 5 and 6 reflect that?

Reply:
yes that is right. I failed to read the diff carefully enough to infer that. thanks you.